node/internal/actors/control_api.go

```diff
@@ -347,14 +347,12 @@ func (api *ControlAPI) handleAgentPing(m *nats.Msg) {
 	var workloadType, namespace, workloadID string
 
 	switch len(splitSub) {
-	case 4: // PREFIX.APING.<WORKLOAD_TYPE>.<NAMESPACE>
-		workloadType = splitSub[2]
-		namespace = splitSub[3]
+	case 4: // PREFIX.APING.<NAMESPACE>.<WORKLOAD_TYPE>
+		namespace = splitSub[2]
+		workloadType = splitSub[3]
```

Comment:
we should probably allow a keyword like `all` for the workload type to return all workloads within a given namespace. Either that or support 2 different token lengths: `prefix.aping.namespace` and `prefix.aping.namespace.all`

I think `prefix.aping.namespace` is more idiomatic NATS

Reply:
ah yes, this is in a different branch I have locally..... essentially mapping `nex workloads ls` to a `$NEX.APING.NAMESPACE` request.  Will do that PR soon